File: content/snippet/code-signal/arcade/intro/island-of-knowledge/array-maximal-adjacent-difference/array-maximal-adjacent-difference.py
def array_maximal_adjacent_difference(inputArray):
    md = 0
    for i in range(len(inputArray) - 1):
        md = max(md, abs(inputArray[i] - inputArray[i + 1]))
    return md


print(array_maximal_adjacent_difference([9, 2, 0, 5]))

File: content/snippet/code-signal/arcade/intro/island-of-knowledge/array-maximal-adjacent-difference/array-maximal-adjacent-difference.js
function arrayMaximalAdjacentDifference(inputArray) {
  let max = 0;
  for (let i = 0; i < inputArray.length - 1; i++) {
    const diff = Math.abs(inputArray[i] - inputArray[i + 1]);
    if (diff > max) {
      max = diff;
    }
  }
  return max;
}

console.log(arrayMaximalAdjacentDifference([9, 2, 0, 5]));
